### PR TITLE
Feature/hfm function

### PR DESCRIPTION
**Pull request by jjy9331**
hiden function menu 
> toggle 
> other space click escape

### CODE REVIEW

File: pfjjy/pages/_document.js [hunk 1]
```diff
@@ -1,19 +1,12 @@
...
 export default function Document() {
   return (
     <Html lang="ko">
       <Head />
       <body>
-        <Header />
         <Main />
         <NextScript />
-        <Hfm />
-        <Footer />
```
Comment (jjy331) [inline pfjjy/pages/_document.js:16]: not working react hook in this file 
> Move to index.js

File: pfjjy/pages/footer.js [hunk 2]
```diff
@@ -1,37 +1,66 @@
...
+                        <li data-descr="more" >
+                            <span className="pf-nav5" data-value="204717"></span>
+                        </li>
+                    </ul>
                 </div>
-            </button>
-    
-        </div>
+                <button className="hfm_btn" onClick={(event) => {
+                    event.stopPropagation();
```
Comment (jjy331) [inline pfjjy/pages/footer.js:48]: bubbling issue solution

**Comment by jjy9331** [comment]
Check point issue
> hook not working in _document.js

**Review by jjy331 — COMMENTED** [review]
Issue & solution

**Pull request merged**